### PR TITLE
🤦 Fix seasonality error

### PR DESCRIPTION
**Pull request by code-arman**
## Status:
:rocket: Ready

## Description

I made a rookie mistake 😞

### CODE REVIEW

**Review by laurenho025 — APPROVED** [review]
very good work

**Review by laurenho025 — CHANGES_REQUESTED** [review]
I retract my approval. Keep the helper for readability but just put locationData && to make sure its defined.

File: src/utils/locationInfo.js [hunk 1]
```diff
@@ -56,4 +56,16 @@ const getStreetAddress = async (lat, lng) => {
   return results[0].formatted_address
 }
 
-export { getFormattedLocationInfo, getLocationInfo, getStreetAddress }
+/**
+ * Determines if seasonality information should be rendered
```
Comment (SirajChokshi) [inline src/utils/locationInfo.js:60]: Maybe make this comment more descriptive of the function rather than its use context outside this file.

**Review by SirajChokshi — APPROVED** [review]
Great work Arman! Really makes up for your first blunder, left a comment about some of your documentation.

**Review by laurenho025 — APPROVED** [review]
great work!

**Pull request merged**